sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -44,11 +45,13 @@ private class CodeFormatter {
     } else {
       indentString
     }
+    code.append(f"${currentLine}%03d ")
```

Comment:
as discussed offline, we can add `/* ... */` to still enable pasting this into an IDE.